R/correctionsataglance-data.R

```diff
@@ -117,12 +117,23 @@ formatDateRange <- function(startD, endD){
 }
 
 formatDataList <- function(dataIn, type, ...){
+  
```

Comment:
this is code I wrote, and I'm definitely open for improvements! I agree with @mhines-usgs: since this is working, additional work should go into a technical debt ticket.